common/persistence/cassandraPersistence.go

```diff
@@ -2011,7 +2017,7 @@ func (d *cassandraPersistence) GetTasks(request *GetTasksRequest) (*GetTasksResp
 		rowTypeTask,
 		request.ReadLevel,
 		request.MaxReadLevel,
-		request.BatchSize)
+	).PageSize(request.BatchSize)
```

Comment:
Let's leave this one to how it was before using the LIMIT.  Matching engine already protects range query by limiting the range of task_id.